README.md

```diff
@@ -334,58 +295,45 @@ julia> SumTypes.show_sumtype(io::IO, ::MIME"text/plain", x::Fruit2) = @cases x b
 julia> apple
 apple!
 ```
-If you overload `Base.show` directly inside a package, you might get annoying method deletion warnings during pre-compilation.
+If you overload `Base.show** directly inside a package, you might get annoying method deletion warnings during pre-compilation.
```

Comment:
Ugh meant for that to be a comment, not a review. Excuse me while I finish waking up.